.github/workflows/website-configs.yml

```diff
@@ -49,8 +49,17 @@ jobs:
       if: "steps.composer-cache.outputs.cache-hit != 'true'"
       run: "composer install --prefer-dist --no-progress --no-suggest --no-dev"
 
-    - name: "Prepare Webite files"
-      run: "bin/console --env=test sync-repositories && bin/console --env=test sync && bin/console --env=test clear-build-cache  && bin/console --env=test build-website-data"
+    - name: "Checkout repositories"
+      run: "bin/console --env=test sync-repositories"
+
+    - name: "Sync repositories"
+      run: "bin/console --env=test sync"
```

Comment:
@SenseException I initially wanted to fix the typo in "website", but then noticed this long line and decided it would be better to split it.

Then I noticed this step. I could not find a command named `sync`, and I think what's actually happening here is Symfony shortcut feature is allowing us to run the previous command (so `sync-repositories`) again. Is this really necessary?